subprojects/provider/src/test/kotlin/org/gradle/kotlin/dsl/accessors/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -171,13 +155,29 @@ class ProjectSchemaAccessorsIntegrationTest : AbstractIntegrationTest() {
                 my { name = "kotlin-dsl" }
             """)
 
-            withFile("settings.gradle.kts")
+            withDefaultSettings()
         }
 
         build("tasks", "-Pmy=lib")
         build("tasks", "-Pmy=app")
     }
 
+    private
+    fun FoldersDsl.withPrecompiledPlugins() {
+        withFile("settings.gradle.kts", defaultSettingsScript)
```

Comment:
:x: `withDefaultSettings()` ?

Reply:
Not possible due to the `FolderDsl` context.

Reply:
Ah right, nevermind